### PR TITLE
InputImage upload label, SymbolDetails language

### PR DESCRIPTION
**Pull request by amberleyromo**
Modified InputImage to conditionally show upload text, modify SymbolDetails language.

Associated with issue #57

### CODE REVIEW

File: src/components/InputImage/InputImage.js [hunk 1]
```diff
@@ -19,16 +19,22 @@ function InputImage(props) {
   }
 
   return (
-    <label className="InputImage">
+    <div className="InputImage">
+      <label
+        htmlFor="imageInput"
+        className={"InputImage__label " + (image ? 'uploaded' : '')}>
```
Comment (shayc) [inline src/components/InputImage/InputImage.js]: We use a package named ‘classnames’ to help with dynamic classNames, also by convention state classes should be prefix with ‘is/has’, so in this case ‘is-uploaded’

File: src/containers/Board/SymbolDetails/SymbolDetails.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ export class SymbolDetails extends Component {
       id: shortid.generate(), // todo: not here
       type: 'symbol',
       label: '',
-      text: '',
+      description: '',
```
Comment (shayc) [inline src/containers/Board/SymbolDetails/SymbolDetails.js:28]: ‘description’ symbol property will later be changed to ‘vocalization’ to align with OBF. Issue #35

File: src/containers/Board/SymbolDetails/messages.js [hunk 3]
```diff
@@ -9,13 +9,17 @@ export default defineMessages({
     id: 'cboard.containers.Board.SymbolDetails.editSymbol',
     defaultMessage: 'Edit Symbol'
   },
+  uploadAnImage: {
```
Comment (shayc) [inline src/containers/Board/SymbolDetails/messages.js:12]: I’ll add the missing strings to crowdin.com

File: src/containers/Board/SymbolSearch/SymbolSearch.css [hunk 4]
```diff
@@ -10,6 +10,14 @@
 
 .react-autosuggest__suggestion {
   width: 25%;
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+
+  /* Adds a hyphen where the word breaks */
+  -webkit-hyphens: auto;
+  -ms-hyphens: auto;
```
Comment (shayc) [inline src/containers/Board/SymbolSearch/SymbolSearch.css]: react-scripts uses ‘AutoPrefixer’ package. I think if you remove these we should be fine

**Review by shayc — DISMISSED** [review]
Looks good, just use classnames package and remove vendor-prefixed css

File: src/components/InputImage/InputImage.js (hunk 1 above)
Comment (shayc) [reply src/components/InputImage/InputImage.js]: https://www.npmjs.com/package/classnames

File: src/containers/Board/SymbolDetails/SymbolDetails.js (hunk 2 above)
Comment (amberleyromo) [reply src/containers/Board/SymbolDetails/SymbolDetails.js:28]: To clarify, don't change this now? If so, all changes have been made.

Comment (shayc) [reply src/containers/Board/SymbolDetails/SymbolDetails.js:28]: that'll be done with issue #35

**Pull request merged**